### PR TITLE
Always run OOP process on .net core

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
We've had support for .net core for years now, and it's been the default for almost a year without any issues.  Removing option to disable this capability.  .Net core is the future.  All hail .net core.

### CODE REVIEW

File: src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs [hunk 1]
```diff
@@ -364,7 +364,6 @@ public bool TryFetch(LocalUserRegistryOptionPersister persister, OptionKey2 opti
         {"dotnet_format_on_save", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "FormatOnSave")},
         {"dotnet_enable_full_solution_analysis_memory_monitor", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "Full Solution Analysis Memory Monitor")},
         {"dotnet_code_analysis_in_separate_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOP64Bit")},
-        {"dotnet_enable_core_clr_in_code_analysis_process", new LocalUserProfileStorage(@"Roslyn\Internal\OnOff\Features", "OOPCoreClr")},
```
Comment (genlu) [inline src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs:367]: We have integration test depends on this, we should remove those CI legs as part of the change

[https://github.com/dotnet/roslyn/blob/02888a4571bfb8ce33f29df4a0504c59be68ca5e/eng/build.ps1#L629](https://github.com/dotnet/roslyn/blob/02888a4571bfb8ce33f29df4a0504c59be68ca5e/eng/build.ps1#L629)

**Review by genlu — APPROVED** [review]
:shipit:

File: src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/VisualStudio/Core/Def/Options/VisualStudioOptionStorage.cs:367]: will do.

**Comment by CyrusNajmabadi** [comment]
@jasonmalinowski 	For review when you get back.

**Pull request merged**